src/main/java/com/elixr/poc/service/UserService.java

```diff
@@ -0,0 +1,59 @@
+package com.elixr.poc.service;
+
+import com.elixr.poc.data.User;
+import com.elixr.poc.rest.request.UserRequest;
+import com.elixr.poc.rest.response.ErrorResponse;
+import com.elixr.poc.rest.response.UserResponse;
+import com.elixr.poc.rest.validator.UserValidator;
+import lombok.Builder;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+/* service class for  editing response and save or update new user to database */
+@Service
+@Builder
+public class UserService {
+
+    /*
+     edit the response by reading the request from user
+     */
+    public ResponseEntity<?> editResponse(UserRequest userRequestObject) {
+        UserResponse responseObject;
+        ErrorResponse userErrorResponse;
+        List<String> errorList = UserValidator.builder().build().validate(userRequestObject);
+        HttpStatus httpStatusCode = HttpStatus.OK;
+        if (!errorList.isEmpty()) {
```

Comment:
I would suggest using the apache commons ObjectUtils.isNotEmpty() method instead of this not-empty checking. So the code will be more readable.

Reply:
Now I am using ObjectsUtils.isNotEmpty().